design/resources/resources-design.json

```diff
@@ -1 +1,3 @@
-{"views":{"count_tags":{"map":"function (doc) {\n        for (var i = 0; i < doc.tags.length; i++) {\n          emit(doc.tags[i], 1);\n        }\n      }","reduce":"_count"},"titles":{"map":"function (doc) {\n        emit(doc.title.toLowerCase().trim(), doc.privateFor);\n      }"}}}
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
@chtelmko This is a merge conflict marker.  Merge conflicts should be handled before pushing to GitHub.  In addition this file should not have been changed at all.

This is the second PR with the exact same commits that modify this file.  Did you commit these to your local `master` branch?  Are you creating new branches off of your `master` branch?